src/librustc_typeck/check/method/suggest.rs

```diff
@@ -771,7 +774,28 @@ fn compute_all_traits<'a, 'gcx, 'tcx>(tcx: TyCtxt<'a, 'gcx, 'tcx>) -> Vec<DefId>
                 _ => {}
             }
         }
+
+        let mut cnums: FxHashSet<CrateNum> = FxHashSet::default();
+
+        // Attempt to load all crates that we have `--extern` flags for, this means
+        // we will be able to make suggestions for traits they define. Particularly useful
+        // in Rust 2018 as there aren't `extern crate` lines that import a crate even if it isn't
+        // otherwise used.
```

Comment:
Thanks, I've renamed this. One of the documentation comments for that query mentioned it was intended for use in error reporting and I couldn't find any uses of the function that weren't in an error reporting context, so this seems fine to rename. Can undo this if there's disagreement.

Reply:
Maybe rename this query to `all_suggestible_traits`, to avoid it getting used in some part of the trait system or something.

cc @rust-lang/compiler